### ISSUE TITLE
CVE-2024-1728 regex problem

### ISSUE THREAD

**Issue by Phoenix1112** [projectdiscovery/nuclei-templates#10509]
It gives incorrect results because more than one regex is used.

```
        regex:
          - "root:.*:0:0:"
          - "\\[(font|extension|file)s\\]"
```

Naturally, this part causes us to get wrong results. ```"\\[(font|extension|file)s\\]"```

**Comment by princechaddha** [comment]
Hello, the response time for this issue was longer than usual because the team was traveling for DEFCON. The team will respond to this issue shortly. Thank you for your contribution

**Comment by DhiyaneshGeek** [comment]
Hi @Phoenix1112 

i have fixed the matcher and raised PR #10652

Let me know if it works 

Thanks for flagging this issue !
